lib/perf_hooks.js

```diff
@@ -395,12 +395,14 @@ class Performance {
 
   measure(name, startMark, endMark) {
     name = `${name}`;
-    endMark = `${endMark}`;
-    startMark = startMark !== undefined ? `${startMark}` : '';
     const marks = this[kIndex][kMarks];
-    if (!marks.has(endMark) && !(endMark in nodeTiming)) {
-      throw new ERR_INVALID_PERFORMANCE_MARK(endMark);
+    if (arguments.length === 3) {
```

Comment:
What if `performance.measure('A to not exists', 'A', 404, 'bar'...)` etc., that `argument.length` is greater than 3 but end mark doesn't exist or is not string-typed?

Reply:
Oh, right. I change to `>= 3`